module3/intermission_work/index.md

```diff
@@ -1,4 +1,81 @@
 ---
 layout: page
 title: Module 3 - Intermission Pre-Work
----
\ No newline at end of file
+---
+
+# Module 3 Intermission Work
+
+## Required Deliverables
+
+### You must submit all of these assignments. *They are due the Saturday before the start of the inning by 5pm*
+
+Submit your work here: [Survey Link](https://forms.gle/p2VnbLdA3QTVVC8N9)
+
+Versions:
+ - Ruby 3.2.2 (run `ruby  -v`)
+ - Rails 7.1.x (run `rails -v`)
+ - Node > v16 (run `node -v`, you are likely on the latest Node version by using `nvm`)
+
+If you need help updating Ruby or Rails versions, check out [this page](https://backend.turing.edu/module3/misc/ruby_and_rails_versions). If you need to revisit Node installation, check out [this page](https://mod0.turing.edu/computer-setup#install-node). 
+
+### 1. Technical Resume
+
+To start your Professional Development work in mod 3, plan to spend 60-90 minutes completing [this async lesson on crafting your Resume](https://docs.google.com/presentation/d/1HDtaMhRxM226YZWQqe93ILUpBIzDLzXFMEKOmjT1yUc/edit#slide=id.p1). 
+
+### 2. Consuming APIs with Ruby
+
+Complete [this exercise](../lessons/consuming_an_api_ruby.md) to gain familiarity with consuming an external API within your Set List application! 
+
+### 3. Read / Take Notes
+[Sandi Metz's _Rules for Developers_](https://robots.thoughtbot.com/sandi-metz-rules-for-developers)
+
+### 4. Practice with Deeply Nested Collections
+
+As we know, when consuming APIs, the data is often returned in deeply nested collections, so when using Ruby, we will need to tap into our Mod 1 skills to practice digging through them to retrieve the data you need.
+
+Fork and clone [Here Be Dragons](https://github.com/turingschool-examples/here-be-dragons). Get the tests to pass. This is something you'll turn in with your submission, so make sure you make a forked copy of it. 
+
+### 5. Service-Oriented Architecture
+
+Review this lesson on Service-Oriented Architecture [here](../lessons/intro_to_soa.md)
+
+### 6. Intro to React
+
+<!-- Watch this video about the React framework. COULDNT FIND A GOOD VIDEO THAT ISNT TOO LONG. Ideabox should have a little intro to React, no? -->
+
+<!-- To-Do: add link for Ideabox -->
+Complete this IdeaBox exercise.
+
+<section class="dropdown">
+## Optional Exercises and Exploration
```

Comment:
The formatting for the optional exercises is a bit weird - I think if we knock each of the headers up one level (or down?) it will make it a little cleaner: 

```
### Optional Exercises and Exploration

#### 1. Additional SQL Practice

#### 2. HTTP Request/Response Cycle Review
* On one piece of paper, write out all of the parts of an example `HTTP GET` request (Diagram the DNS look-up as well as how a Rails Application would handle the request via MVC)
  * **Even better:** write your explanation as a metaphor
* On a separate piece of paper, write out an example 200 response to that request with all of the parts

#### 3. Data Structures and Algorithms

Read: [An Overview of Data Structures for Ruby Developers](https://www.rubyguides.com/2019/04/ruby-data-structures/)

We are going to be covering various data structures to prepare you for the job hunt and technical interviews. Complete this former M1 project, [Beat Box](https://backend.turing.edu/module1/projects/beat_box).

If you've done Beat Box, let's get serious and have a [Date Night](https://backend.turing.edu/module1/projects/date_night)!

#### 4. Security

* Check out [Hacksplaining](https://www.hacksplaining.com/lessons) and go through these lessons, or whatever seems interesting to you!
  * SQL Injection
  * Cross-Site Request Forgery
  * Crros-Site Scripting
  * Password Mismanagement

* Read the docs on [Rails Security](https://guides.rubyonrails.org/security.html). We recommend focusing in particular on the sections regarding:
  * Sessions
  * CSRF
  * User Management
  * Injection
```

Reply:
Sounds good to me!